src/plugins/janus_videoroom.c

```diff
@@ -7276,6 +7322,8 @@ static json_t *janus_videoroom_process_synchronous_request(janus_videoroom_sessi
 		recipient->port = port;
 		recipient->rtcp_port = rtcp_port;
 		recipient->rtcp_added = rtcp_added;
+		recipient->srtp_suite = srtp_suite;
+		recipient->srtp_crypto = g_strdup(srtp_crypto);
```

Comment:
I can't remember if glib complains if you pass `NULL` to `g_strdup`. At any rate, for a matter of consistency with the rest of the code, it might be better to use a ternary here, e.g.

    recipient->srtp_crypto = srtp_crypto ? g_strdup(srtp_crypto) : NULL;